cmd/juju/ssh/ssh_jump.go

```diff
@@ -50,6 +55,13 @@ type sshJump struct {
 	hostChecker            jujussh.ReachableChecker
 	publicKeyRetryStrategy retry.CallArgs
 	jumpHostPort           int
+	showCommand            bool
+
+	outputTemplate *template.Template
+}
+
+func (p *sshJump) SetFlags(f *gnuflag.FlagSet) {
+	f.BoolVar(&p.showCommand, "show-command", false, "Print the OpenSSH command instead of executing it. It is enabled only with the --jump flag.")
```

Comment:
Maybe 
```suggestion
	f.BoolVar(&p.showCommand, "show-command", false, "Print the OpenSSH command instead of executing it. This flag is only enabled with the --jump flag.")
```